hooks/tk-multi-publish2/maya.basic/publish_maya_session.py

```diff
@@ -197,11 +197,13 @@ def validate(self, settings, item):
         if not path:
             # the session still requires saving. provide a save button.
             # validation fails.
+            session_error_message = "The Maya session has not been saved."
             self.logger.error(
-                "The Maya session has not been saved.",
+                session_error_message,
                 extra=_get_save_as_action()
             )
-            return False
+            # this exception should be cought by plugin.py run_validate() and propagate to tree_node_task.py validate()
+            raise Exception(session_error_message)
```

Comment:
The exception is first caught in [run_validate()](https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/processing/plugin.py#L225) then in  
[validate()](https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/publish_tree_widget/tree_node_task.py#L111)